release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -268,10 +269,16 @@ class RepoInfoExpansionState extends State<RepoInfoExpansion> {
                       decoration: const BoxDecoration(border: Border(top: BorderSide(color: Colors.grey))),
                       children: <Widget>[
                         Text('$repoElement:'),
-                        SelectableText(
-                            (widget.currentStatus[repoElement] == null || widget.currentStatus[repoElement] == '')
-                                ? 'Unknown'
-                                : widget.currentStatus[repoElement]! as String),
+                        repoElement == ConductorStatus.engineRepoElements[4] ||
```

Comment:
Avoid using magic numbers. Prefer assigning enums to the values

Reply:
Could you provide an example of how to assign enums to these, please?

Reply:
You could do something like

```
enum EngineRepoWidgets {
  firstKey = 1;
  ...
  EngineTestConsole = 4;
}
```

Note, the `firstKey` should be descriptive to what it's related to.